web/src/lib/components/files/FileActions.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+import type { File } from "$lib/api";
+import { createDropdownMenu, melt } from "@melt-ui/svelte";
+import { Localized } from "@nubolab-ffwd/svelte-fluent";
+import { createEventDispatcher } from "svelte";
+
+type Actions = "rename" | "download" | "delete";
+
+interface Action {
+	icon: `fa-${string} fa-${string}`;
+	action: Actions;
+	disabled?: true;
+}
+
+const actions: Action[] = [
+	{ action: "rename", icon: "fa-solid fa-pencil", disabled: true },
+	{ action: "download", icon: "fa-solid fa-download" },
+	{ action: "delete", icon: "fa-solid fa-trash", disabled: true },
+];
+
+const dispatch = createEventDispatcher<Record<Actions, File>>();
+
+let clazz = "";
```

Comment:
what is that?

Reply:
`class` is a reserved keyword in JavaScript/TypeScript, used to define new classes.
```js
class Hello {
}
```

To be able to expose an HTML property called `class` (to add CSS classes to a component), a neat trick in Svelte is to define it like this:

```svelte
<script>
let clazz = "";
export { clazz as class };
</script>

<MyComponent class="" />
```
This way, the internal variable (usually called `clazz`, but could as well be called `className` or `pippo`) is exported as `class` without clashing with the keyword.